spec/models/card_spec.rb

```diff
@@ -1,7 +1,9 @@
 require "rails_helper"
 
 describe Card do
-  let(:card) { FactoryGirl.create :card, translated_text: " Correct value " }
+  let(:user) { FactoryGirl.create :user }
+  let(:card) { FactoryGirl.create :card, translated_text: " Correct value ",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.